package.json

```diff
@@ -44,7 +44,7 @@
     "jquery": "^2.1.4",
     "moment": "^2.10.3",
     "moment-business": "^2.0.0",
-    "node-sass": "3.4.2",
+    "node-sass": "^3.7.0",
```

Comment:
If we are using hat character, we could just specify "3.0.0" to get latest minor or patch version?